backend/daemon/storage/migrations.go

```diff
@@ -50,6 +50,38 @@ var migrations = []migration{
 	{Version: "2023-07-12.01", Run: func(d *Dir, conn *sqlite.Conn) error {
 		return sqlitex.ExecScript(conn, "PRAGMA user_version = 0;")
 	}},
+	//Sites migration Hotfix
+	{Version: "2023-07-25.01", Run: func(d *Dir, conn *sqlite.Conn) error {
+		return sqlitex.ExecScript(conn, `
+PRAGMA foreign_keys=off;
+
+BEGIN TRANSACTION;
```

Comment:
Does this work when you try it locally? Because all migrations are already wrapped in a transaction. This should fail because of a nested transaction.

Reply:
You're right now it works locally, tested with the failing database